account_asset_management/models/account_asset.py

```diff
@@ -325,7 +327,15 @@ def _compute_move_line_check(self):
         for asset in self:
             asset.move_line_check = bool(
                 asset.depreciation_line_ids.filtered("move_id")
+                and not asset.depreciation_line_ids.filtered(
+                    lambda l: l.type == "create"
```

Comment:
Actually, I am not sure we can change this field because it seems normal not to be able to change the purchase value if it comes from the invoice.
Maybe you would need to had a field like "has_depreciation_accounting_entry" that check if at least one depreciation lines of type `depreciate` has an accounting entry, and use this new field for the readonly of `total_number_of_use` ?
And keep the existing `move_line_check` for the `purchase_value`

Also, this condition seems wrong, because it checks if there is a line of type create, it does not check if it is linked to an `account.move`.